bin/hbase

```diff
@@ -796,10 +799,6 @@ HBASE_OPTS="$HBASE_OPTS -Dhbase.log.file=$HBASE_LOGFILE"
 HBASE_OPTS="$HBASE_OPTS -Dhbase.home.dir=$HBASE_HOME"
 HBASE_OPTS="$HBASE_OPTS -Dhbase.id.str=$HBASE_IDENT_STRING"
 HBASE_OPTS="$HBASE_OPTS -Dhbase.root.logger=${HBASE_ROOT_LOGGER:-INFO,console}"
-if [ "x$JAVA_LIBRARY_PATH" != "x" ]; then
```

Comment:
Does dropping this hunk affect native Hadoop library support? 
I didn't check if this is just a removal of something redundant.

Reply:
I can not recall why I dropped this when migrating to log4j2 on master branch...

Will dig more and report back later.

Reply:
Seems to be a cherry-pick mistake, we stll have this on master. Restored it back.